src/Uno.Extensions.Authentication.MSAL/MsalAuthenticationProvider.cs

```diff
@@ -50,7 +51,12 @@ public void Build()
 		}
 
 #if WINDOWS
-		builder.WithWindowsEmbeddedBrowserSupport();
+		builder.WithBroker(new BrokerOptions(BrokerOptions.OperatingSystems.Windows));
+		builder.WithParentActivityOrWindow(() =>
+		{
+			IntPtr hwnd = WindowNative.GetWindowHandle(window);
```

Comment:
Actually you coudl remove the `using WinRT.Interop` and just fully qualify this to `WinRT.Interop.InitializeWithWindow` like we do here: https://github.com/unoplatform/uno.extensions/blob/78184f87453578f3fa6f9a9b1275ada1ef7b7fe6/src/Uno.Extensions.Navigation.UI/Navigators/MessageDialogNavigator.cs#L101